docs/sources/reference/run.md

```diff
@@ -552,6 +553,22 @@ would be 2*300M, so processes can use 300M swap memory as well.
 We set both memory and swap memory, so the processes in the container can use
 300M memory and 700M swap memory.
 
+By default, processes in a container are killed if an out-of-memory (OOM) occurs.
+To disable this behaviour, use the `--oom-kill-disable` option. Disabling the OOM killer
+should only be done on containers that have the `-m/--memory` option set.
+
+Examples:
+
+    $ docker run -ti -m 100M --oom-kill-disable ubuntu:14.04 /bin/bash
```

Comment:
It's because of the (IMO) confusing way that Boolean flags are documented. Basically, `--enable-foo=true` is the same as `--enable-foo`, and `--enable-foo=false` is the same as **not** setting the `--enable-foo` flag. For more info, see https://github.com/docker/docker/issues/9406.

In general, you don't actually add the true/false value, even though you *can*.
If you're okay with not documenting the `=true|false` here, I'd be +1, but it would be inconsistent with other parts of the docs 😞